tlspuffin/src/protocol.rs

```diff
@@ -846,6 +925,8 @@ pub struct TLSDescriptorConfig {
     pub try_reuse: bool,
     /// List of available TLS ciphers
     pub cipher_string: String,
+    /// List of available TLS groups/curves
+    pub groups: Option<String>,
```

Comment:
By default this is None, so please modify the comment to explain what this field will do then (overwrite the PUT default?)

Reply:
Done